### PR TITLE
Remove unnecessary documentation

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/soap/soapserver.xml [hunk 1]
```diff
@@ -34,11 +34,6 @@
     </classsynopsisinfo>
 
     <classsynopsisinfo role="comment">&Properties;</classsynopsisinfo>
-    <fieldsynopsis>
```
Comment (kocsismate) [inline reference/soap/soapserver.xml:37]: As far as I saw, there is no such property (not even a dynamic one). Even if there is one, it's private, so it's not a big loss not to document it.

File: reference/spl/splstack.xml [hunk 2]
```diff
@@ -41,11 +41,6 @@
      <xi:fallback/>
     </xi:include>
 
-    <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
```
Comment (kocsismate) [inline reference/spl/splstack.xml:44]: Only `SplStack::setIteratorMode()` is listed in this section, which is an inherited method from `SplDoublyLinkedList`. The latter file has all the necessary information for SplStacks (with a little bit of addition).

**Review by cmb69 — APPROVED** [review]
Thank you! Looks good to me (besides the minor wording issue).

File: reference/spl/spldoublylinkedlist/setiteratormode.xml [hunk 3]
```diff
@@ -44,6 +44,13 @@
       <para>
        The default mode is: <constant>SplDoublyLinkedList::IT_MODE_FIFO</constant> | <constant>SplDoublyLinkedList::IT_MODE_KEEP</constant>
       </para>
+
+      <warning>
+       <para>
+        The direction of iteration can no longer be changed for <classname>SplStack</classname>s.
```
[the remark this answers is not in the document]
Comment (kocsismate) [reply reference/spl/spldoublylinkedlist/setiteratormode.xml]: Nice catch! I've recently pushed a new commit to fix that as well.

**Pull request merged**